templates/authStep2.tpl

```diff
@@ -156,9 +142,9 @@
 			</fieldset>
 			{* Allow the user to sign up as a reviewer *}
 			{if isset($currentContext) }
-			{assign var=contextId value=$currentContext->getId()}
+				{assign var=contextId value=$currentContext->getId()}
 			{else}
-			{assign var=contextId value=0}
+				{assign var=contextId value=0}
```

Comment:
Use a constant rather than a magic number.

Reply:
This is changed in `stable-3_4_0` branch - will be picked-up on main branch after merging the stable and porting to main.